src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import command.Command;
+import main.*;
```

Comment:
I think maybe can try to import the specific task

Reply:
Thanks for pointing it out 👍